medusa/helper/common.py

```diff
@@ -205,9 +205,9 @@ def convert_size(size, default=None, use_decimal=False, **kwargs):
             scalar, units = size_tuple[0], size_tuple[1:]
             units = units[0].upper() if units else default_units
         else:
-            regex_units = re.search(r'(\w+)', size, re.IGNORECASE)
-            units = regex_units.group() if regex_units else default_units
-            scalar = size.strip(units)
+            regex_units = re.search(r'(\d*[\\.|\\,]?\d*)(\s?({scale}))'.format(scale='|'.join(scale)), size, re.IGNORECASE)
```

Comment:
Okay sure, then.. `([\d.]+)(\s?(B|KB|MB|GB|TB|PB))` ?
As we shouldn't need to take into account the comma then?